### DISCUSSION TITLE
Treatment untuk variabel didataset yang tidak memiliki korelasi

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#474] [Q&A] [answered]
Dear rekan2 Algoritma,

Mohon saran dan tanggapannya jika kita memiliki dataset yang korelasinya mayoritas independent, treatmentnya seperti apa ya ?
Berikut heat map korelasi antar data.
![Korelasi](https://user-images.githubusercontent.com/103427989/206411844-dc4908f9-0a6a-45a2-8cc7-2e8a61110604.jpg)

**Comment by nisa-basalamah** [comment]
Dear @joejnt,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima. Kami akan mencoba membantu mencarikan referensi berkaitan dengan pertanyaan yang diberikan.

**Reply by joejnt** [reply]
@nisa-basalamah 

Terima kasih untuk supportnya, adapun case yang akan saya gunakan adalah untuk _clustering_

**Comment by kevbow** [comment, marked as answer]
Dear @joejnt,

Kasus clustering tidak mewajibkan Anda untuk memiliki data yang berkorelasi sehingga Anda bisa langsung menggunakan datanya tanpa melakukan treatment. 
Demikian dan semoga membantu.

**Reply by joejnt** [reply]
Dear @kevbow 

Terima kasih untuk inputnya, kalo untuk kasus classification gimana ya treatmentnya, apakah kita perlu ganti variabelnya atau ada treatmentnya.

**Reply by kevbow** [reply]
Dear @joejnt ,

Tergantung dari model machine learning yang digunakan. Pada metode Logistic Regression, ada aturan linearity antara prediktor dengan log of odds dari targetnya, tetapi selain metode tersebut tidak ada yang mewajibkan adanya hubungan linearity atau korelasi. 
Tidak ada treatment spesifik untuk kasus ini selain menambahkan data maupun melakukan feature engineering. Berikut referensi yang dapat kami berikan dengan kasus yang mungkin serupa:
- https://www.kaggle.com/questions-and-answers/102805

Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by joejnt** [reply]
well notes @kevbow ..terima kasih untuk insight nya
